samples/core/src/main/resources/db/migration/mysql/V0002__R001_Master_data_MYSQL.sql.mysql

```diff
@@ -50,6 +50,6 @@ INSERT INTO BILLORDERPOSITION (billId, orderPositionsId) VALUES (2,3);
 INSERT INTO BILLORDERPOSITION (billId, orderPositionsId) VALUES (2,4);
 
 INSERT INTO STAFFMEMBER (id, modificationCounter, role, login, firstname, lastname) VALUES (1, 0, 3, 'chief', 'Charly', 'Chief');
-INSERT INTO STAFFMEMBER (id, modificationCounter, role, login, firstname, lastname) VALUES (2, 0, 0, 'cook', 'Carl', 'Cook');
+INSERT INTO STAFFMEMBER (id, modificationCounter, role, login, firstname, lastname) VALUES (2, 0, 1, 'cook', 'Carl', 'Cook');
```

Comment:
is this change correct? the value `1` is not unique in this modification... but it was previously